jwt/src/commonMain/kotlin/com/appstractive/jwt/JWT.kt

```diff
@@ -49,8 +50,7 @@ data class JWT(
   }
 
   override fun hashCode(): Int {
-    var result = header.hashCode()
-    result = 31 * result + claims.hashCode()
+    var result = signedData.hashCode()
```

Comment:
`header` and `claims` should still be part of the hash. Even though `signedData` technically contains them both, one could create a copy of the JWT via `copy()` and change some of them. That new JWT object would then have the same hash as the old one, but have different values, possibly leading to undesired consequences.

Reply:
I added the header and claims again to the hashCode calculation.